codesy/templates/includes/faq.html

```diff
@@ -68,6 +68,10 @@
                <dt>What happens if more money is offered than some asks?</dt>
                <dd><p>
                    The amount of extra money is evenly split between the offerors and claimant. We think this fairly adds a bonus to the claimant and a discount for the offerors.
+                   For example, if a dev asks for $5 and there are two offers of $5 and $10 the extra $15 will be split between the three participants.
+                   The dev will recieve $10 (minus processing fees) instead of $5.
+                   The 
```

Comment:
The .... ? The what?! The suspense is killing me!!

![kp1ogdaax05oi](https://cloud.githubusercontent.com/assets/71928/25066909/35b78584-21f9-11e7-948a-a286642a634a.gif)